upb/msg_internal.h

```diff
@@ -198,6 +198,18 @@ typedef struct {
   int ext_count;
 } upb_msglayout_file;
 
+// Computes a bitmask in which the |n| lowest bits are set, except that we
```

Comment:
the comment says `n` but that is a derived quantity.  You should make it say `l->required_count`

Reply:
Done.